extension-androidauto/src/main/AndroidManifest.xml

```diff
@@ -1 +1,6 @@
-<manifest package="com.mapbox.maps.extension.androidauto" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mapbox.maps.extension.androidauto">
+
+    <uses-permission android:name="androidx.car.app.ACCESS_SURFACE" />
```

Comment:
is this the only permission the Android Auto extension needs?

Reply:
In order to create a screen that supports the permission, you would need to use a NavigationTemplate. Which requires the `<uses-permission android:name="androidx.car.app.NAVIGATION_TEMPLATES" />` permission.

I have been on the fence some here. The SDK does not actually use any templates, so I left the permission out.

But here is a good reason to include the NAVIGATION_TEMPLATE. We were denied from the play store when we included the `<uses-permission android:name="androidx.car.app.MAP_TEMPLATES"/>`  permission. https://issuetracker.google.com/u/0/issues/193457334

Another reason against adding the permission, developers are asking for MAP_TEMPLATES to support the ACCESS_SURFACE https://issuetracker.google.com/u/0/issues/171715013#comment7

So I've defaulted to, including less permissions for now.

Reply:
IMHO lets stick to the option that allows to publish without rejections. If I understood corectly it's including `NAVIGATION_TEMPLATES`, right? Ticket https://issuetracker.google.com/u/0/issues/193457334 is a bit confusing...

Reply:
Oh it was the `category` that caused the google play rejection. So this permission does not impact that. I don't think our SDK should have a default implementation for this, considering this is specific to android auto car library

``` xml
        <service
            android:name=".service.MapboxCarAppService"
            android:exported="true"
            tools:ignore="ExportedService">

            <intent-filter>
                <action android:name="androidx.car.app.CarAppService" />
                <category android:name="androidx.car.app.category.NAVIGATION" />
            </intent-filter>
        </service>
```

So my reason above is incorrect (i was also confused by my old issuetracker because we learned of the problem later, in the comments).

Reply:
The reason we'd add `NAVIGATION_TEMPLATES` is to simplify a developers setup very slightly. 

But i'd also argue it is confusing because they will be following the directions here https://developer.android.com/training/cars/apps/navigation

Considering this issue minor, whatever we do just needs to be reflected in developer documentation. But I'm also fine with either way. @yunikkk let me know if you're passionate one way or another or i'll leave it as is

Reply:
@kmadsen I think it's ok to leave it as is, however would like to highlight the other permission required in the README file.

Reply:
Just noticed this. Yeah thanks for flagging, @pengdev. Updated the README